app/views/incident_reports/_form.html.haml

```diff
@@ -205,6 +209,7 @@
       .field
         = form.label :police_case_assigned
         = form.text_field :police_case_assigned, id: :report_police_case_assigned
+        .example 17-123-AC
```

Comment:
This is nice. Why didn't you put the examples into the text fields?

Reply:
As an actual placeholder? That was what I suggested, but it was thought that people might confuse the placeholder for having already filled in the field.

Reply:
Right. And okay, not your design choice. I won't argue with you about this then.